README.md

```diff
@@ -2,6 +2,7 @@
 [![Build Status](https://travis-ci.org/Khan/KaTeX.svg?branch=master)](https://travis-ci.org/Khan/KaTeX)
 [![codecov](https://codecov.io/gh/Khan/KaTeX/branch/master/graph/badge.svg)](https://codecov.io/gh/Khan/KaTeX)
 [![Join the chat at https://gitter.im/Khan/KaTeX](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/Khan/KaTeX?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+![](https://img.badgesize.io/Khan/KaTeX/v0.9.0/dist/katex.min.js?compression=gzip)
```

Comment:
It seems that `badgesize` doesn't support https.

Reply:
There seems to be two alternative:
(i) use `shields.io` file size badge: https://img.shields.io/github/size/Khan/KaTeX/dist/katex.min.js%3Fref%3Dv0.9.0.svg (Note: the `ref` part is not an official API)
(ii) calculate file sizes in release script and create custom badge: e.g., https://img.shields.io/badge/size-234%20kb-brightgreen.svg

Reply:
I think we can use the one from shields.io